docs/public/specifications/tests/Zone-TP/zone11.md

```diff
@@ -41,13 +41,13 @@ server.
 
 Message Tag                      | Level   | Arguments    | Message ID for message tag
 :--------------------------------|:--------|:-------------|:--------------------------------------------
-Z11_INCONSISTENT_SPF_POLICIES    | WARNING |              | The *Child Zone* publishes different SPF policies on different name servers.
-Z11_DIFFERENT_SPF_POLICIES_FOUND | NOTICE  | ns_ip_list   | The following name servers returned the same SPF version 1 policy, but other name servers returned a different policy. Name servers: {ns_ip_list}.
-Z11_NO_SPF_FOUND                 | NOTICE  |              | The *Child Zone* does not publish an SPF policy.
-Z11_SPF1_MULTIPLE_RECORDS        | ERROR   | ns_ip_list   | The following name servers returned more than one SPF version 1 policy. Name servers: {ns_ip_list}.
-Z11_SPF1_SYNTAX_ERROR            | ERROR   | ns_ip_list   | The *Child Zone*’s SPF version 1 policy has a syntax error. Policy retrieved from the following nameservers: {ns_ip_list}.
-Z11_SPF1_SYNTAX_OK               | INFO    |              | The *Child Zone*’s SPF version 1 policy has correct syntax.
-Z11_UNABLE_TO_CHECK_FOR_SPF      | ERROR   |              | None of the name servers for *Child Zone* responded with an authoritative response to queries for SPF policies.
+Z11_INCONSISTENT_SPF_POLICIES    | WARNING |              | The zone publishes different SPF policies on different name servers.
+Z11_DIFFERENT_SPF_POLICIES_FOUND | NOTICE  | ns_ip_list   | The following name servers returned the same SPF policy, but other name servers returned a different policy. Name servers: {ns_ip_list}.
+Z11_NO_SPF_FOUND                 | NOTICE  |              | The zone does not publish an SPF policy.
+Z11_SPF1_MULTIPLE_RECORDS        | ERROR   | ns_ip_list   | The following name servers returned more than one SPF policy. Name servers: {ns_ip_list}.
+Z11_SPF1_SYNTAX_ERROR            | ERROR   | ns_ip_list   | The zone’s SPF policy has a syntax error. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_SYNTAX_OK               | INFO    |              | The zone’s SPF policy has correct syntax.
+Z11_UNABLE_TO_CHECK_FOR_SPF      | ERROR   |              | None of the zone’s name servers responded with an authoritative response to queries for SPF policies.
```

Comment:
To be strict, the policy is a feature of a name (mail domain) not a zone. In a zone you could have multiple SPF policies for different mail domains, can't you. E.g. in the zone `test.xa` you could have one for `afnic.test.xa` and another for `iis.test.xa` given that those are used for mail addresses, e.g. `info@afnic.test.xa` and `info@iis.test.xa`, respectively. Zonemaster only tests the SPF policy of the mail domain equal to the zone apex, but that is still just a name.

I suggest something like "The SPF policy of the zone apex (...)"

(I also put the same comment in https://github.com/zonemaster/zonemaster-engine/pull/1348#pullrequestreview-2069648648)

Reply:
Or maybe something like "The SPF policy of {domain} has (...)" where "{domain}" is the name of apex. It will probably be easier to understand than my first suggestion,

Reply:
I’ll do that. However, adding the `{domain}` argument will require a database migration script in order not to break existing messages, because they do not have any arguments.

Reply:
This is something that we have not considered in the past, but it is not irrelevant. However, if we have that requirement it will be hard to update the messages in the future. When we remove a tag (or rename) it will affect even more.

Reply:
Yes, database migrations are annoying, but I have another PR open that could use a similar script for the same reason. We could piggyback on that one if we want.